apps/opencs/view/render/terrainshapemode.hpp

```diff
@@ -57,6 +58,7 @@ namespace CSVRender
 
             /// Editmode for terrain shape grid
             TerrainShapeMode(WorldspaceWidget*, osg::Group* parentNode, QWidget* parent = nullptr);
+            ~TerrainShapeMode();
```

Comment:
This should have a virtual. Not strictly needed, but helps with clarity.

Reply:
Removed it altogether in 66fba7c .